bs.py

```diff
@@ -207,14 +204,14 @@ def store_hit(board, ships, guess):    ##TEST THIS BEFORE USING - TEST HITS INCR
 def guess(board, ships):
     while True:
         guess = get_guess(board)
-        os.system('clear')
-        if check_guess_legal(board, guess) == True:
-            if check_cell_guessed(board, guess) == False:
-                if check_guess_hit(board, guess) == True:
+        os.system('clear')  # clever
+        if check_guess_legal(board, guess):  # no need to compare against explicit True/False
+            if not check_cell_guessed(board, guess):
+                if check_guess_hit(board, guess):
                     print "Hit!"
                     board, ships = store_hit(board, ships, guess)
                     break
-                elif check_guess_hit(board, guess) == False:
```

Comment:
I had left elif here so that if there were any un-expected errors, it would just reloop the while. Is that bad practice?